spec/features/verification_upload_spec.rb

```diff
@@ -3,10 +3,25 @@
 feature "Uploading verifications", js: true do
   before do
     visit "/"
+    expect(page).to have_text "Report the change"
```

Comment:
Are these spec changes mean to be with this pull request?

Reply:
Yep! As part of this change, we need to make sure we have a member before we skip to where this test skips. Before we didn't. I added some extra text in the test to help us clarify where we are in the flow, too.